docs-content/general/7_integrations/launchdarkly-integration.md

```diff
@@ -0,0 +1,34 @@
+---
+title: LaunchDarkly Integration
+slug: launchdarkly-integration
+createdAt: 2025-04-03T23:57:09.000Z
+updatedAt: 2025-04-03T23:57:09.000Z
+---
+
+We've made it easy to use LaunchDarkly with Highlight.
+
+The integration will allow you to report Feature Flag Evaluations to Highlight
+and to send the LaunchDarkly identification context to your Highlight session.
+
+Conversely, it will hook into the Highlight SDK and send the user context to LaunchDarkly.
+
+```typescript
+H.init("<YOUR_PROJECT_ID>", {});
+const ldClient = initialize("<YOUR_LD_CLIENT_TOKEN>");
+H.registerLD(ldClient);
+```
+
+## API
+
+### `identify()`
+
+Calling [`H.identify()`](../../sdk/client.md#Htrack) will forward the data to LaunchDarkly's `identify()`. 
```

Comment:
We talked offline some, but we may not want to do this.

Reply:
Yeah, I avoided doing that in https://github.com/highlight/highlight/pull/10096/files#diff-8472a4bb462a3f26d4e308c51f935ec34642690e789d5dfee93989bd4fc350ac.

Reply:
Good catch; I had [removed this from the SDK](https://github.com/highlight/highlight/blob/d7ab0fb54dd8f0aed8886d761a0f204427b11240/sdk/client/src/integrations/launchdarkly/index.ts#L124) but forgot to update the doc.